websites/C/cda.pl/presence.ts

```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
+			largeImageKey: "https://i.imgur.com/AD1naGM.png",
 			startTimestamp: browsingTimestamp
 		},
-		pathname: string = document.location.pathname.toLowerCase();
+		path = document.location.pathname.toLowerCase();
 
-	if (pathname === "/") {
-		presenceData.details = "Przegląda:";
-		presenceData.state = "Stronę Główną";
-		presenceData.smallImageKey = "browsing";
-	} else if (
-		pathname.includes("/video") &&
-		document.querySelector("#naglowek > span > span > h1")
-	) {
-		const title = document.querySelector(
-			"#naglowek > span > span > h1"
-		).textContent;
-		presenceData.details = title;
-		presenceData.state = `Przesłał: ${
-			document.querySelector(
-				"#leftCol > div:nth-child(2) > div.DescrVID > div.DescrVID-left > div > div > div > div:nth-child(1) > a > span > span"
-			).textContent
-		}`;
-		if (
-			window
-				.getComputedStyle(
-					document.querySelector(
-						"#player > div > div > div > div > div > div > span.pb-video-player-wrap > span.pb-video-ad-container"
-					),
-					null
-				)
-				.getPropertyValue("display") === "block"
-		) {
-			presenceData.details = `(Reklama) ${title}`;
+	if (path == "/") presenceData.details = "Przegląda stronę główną";
+	else if (path.endsWith("/ustawienia"))
+		presenceData.details = "Przegląda ustawienia";
+	else if (path.endsWith("/poczta")) presenceData.details = "Przegląda pocztę";
+	else if (path.endsWith("/premium"))
+		presenceData.details = "Przegląda filmy CDA Premium";
+	else if (path.endsWith("/powiazanekonta"))
+		presenceData.details = "Przeglądanie powiązanych kont";
```

Comment:
This should be "Przegląda powiązane konta" to be consistent with the other presences.